SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -137,16 +139,37 @@ void FSpatialGDKServicesModule::ExecuteAndReadOutput(const FString& Executable,
 	FPlatformProcess::ClosePipe(0, WritePipe);
 }
 
+void FSpatialGDKServicesModule::SetProjectName(const FString& InProjectName)
+{
+	FString SpatialFileResult;
+
+	if (!JsonParsedSpatialFile)
```

Comment:
This is the file that's parsed at start-up yeah? Wonder if we should make a method to `TryGetSpatrialProjectFile` which re-parses it if parsing initially failed.